src/SIL.XForge.Scripture/ClientApp/.storybook/preview.ts

```diff
@@ -15,8 +15,7 @@ export const parameters = {
       color: /(background|color)$/i,
       date: /Date$/
     }
-  },
-  docs: { inlineStories: true }
```

Comment:
This line was added when migrating to Storybook 7 in order to keep old behavior. It's not needed as I'm turning off autodocs anyway.